### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by sergi0g** [pybluez/pybluez#433]
I am trying to install the package but pip keeps failing. The error I get is this:
---------------------------------------------------------------------------------------------------------------------------------------
Collecting PyBluez
  Using cached PyBluez-0.23.tar.gz (97 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [1 lines of output]
      error in PyBluez setup command: use_2to3 is invalid.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

**Comment by miigotu** [comment]
Same here:
```bash
lsb_release -a:
Distributor ID:	Ubuntu
Description:	Ubuntu Jammy Jellyfish (development branch)
Release:	22.04
Codename:	jammy

uname -a:
Linux RAIDER 5.16.10-051610-generic #202202161442 SMP PREEMPT Wed Feb 16 14:49:20 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

python3 --version
Python 3.9.10

python3 -m pip install pybluez -v:
Using pip 22.0.2 from /usr/lib/python3/dist-packages/pip (python 3.9)
Defaulting to user installation because normal site-packages is not writeable
Collecting pybluez
  Using cached PyBluez-0.23.tar.gz (97 kB)
  Running command python setup.py egg_info
  error in PyBluez setup command: use_2to3 is invalid.
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  full command: /usr/bin/python3 -c '
  exec(compile('"'"''"'"''"'"'
  # This is <pip-setuptools-caller> -- a caller that pip uses to run setup.py
  #
  # - It imports setuptools before invoking setup.py, to enable projects that directly
  #   import from `distutils.core` to work with newer packaging standards.
  # - It provides a clear error message when setuptools is not installed.
  # - It sets `sys.argv[0]` to the underlying `setup.py`, when invoking `setup.py` so
  #   setuptools doesn'"'"'t think the script is `-c`. This avoids the following warning:
  #     manifest_maker: standard file '"'"'-c'"'"' not found".
  # - It generates a shim setup.py, for handling setup.cfg-only projects.
  import os, sys, tokenize
  
  try:
      import setuptools
  except ImportError as error:
      print(
          "ERROR: Can not execute `setup.py` since setuptools is not available in "
          "the build environment.",
          file=sys.stderr,
      )
      sys.exit(1)
  
  __file__ = %r
  sys.argv[0] = __file__
  
  if os.path.exists(__file__):
      filename = __file__
      with tokenize.open(__file__) as f:
          setup_py_code = f.read()
  else:
      filename = "<auto-generated setuptools caller>"
      setup_py_code = "from setuptools import setup; setup()"
  
  exec(compile(setup_py_code, filename, "exec"))
  '"'"''"'"''"'"' % ('"'"'/tmp/pip-install-68qwtmvz/pybluez_c1c9b215cbe7493a9f160507e1e4e9c2/setup.py'"'"',), "<pip-setuptools-caller>", "exec"))' egg_info --egg-base /tmp/pip-pip-egg-info-ew_2egyz
  cwd: /tmp/pip-install-68qwtmvz/pybluez_c1c9b215cbe7493a9f160507e1e4e9c2/
  Preparing metadata (setup.py) ... error
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

```

**Comment by miigotu** [comment]
Related issues (did not check them for solutions yet):
#413 #424 #431

**Comment by miigotu** [comment]
This was fixed in #431 - Solution for now, until a new release is pushed to pypi is:
```bash
pip3 install git+https://github.com/pybluez/pybluez.git#egg=pybluez
```

**Comment by PRO-2684** [comment]
Another approach to fix this issue: https://github.com/pybluez/pybluez/issues/438#issuecomment-1107701984
